engine/kv_engine_sorted.cpp

```diff
@@ -265,6 +265,28 @@ Status KVEngine::SortedPutImpl(Skiplist* skiplist, const StringView& user_key,
   auto ul = hash_table_->AcquireLock(collection_key);
   TimeStampType new_ts = version_controller_.GetCurrentTimestamp();
   auto ret = skiplist->Put(user_key, value, new_ts);
+
+  // Collect outdated version records
+  auto old_record = removeOutDatedVersion<DLRecord>(ret.write_record);
```

Comment:
if(ret.existing_record) {removeOutdatedVersion<DLRecord>(ret.write_record);}
I think this is better, to save time for insert operation